doc/developers/contributing.rst

```diff
@@ -1519,6 +1519,9 @@ non_deterministic
 requires_positive_data - unused for now
     whether the estimator requires positive X.
 
+requires_positive_y
+    whether the estimator requires a positive y (only applicable for regression).
```

Comment:
Maybe we should call the one above `requires_positive_X` to be consistent, not sure. In any case, I am not so keen on `requires_positive_target` if we try to reach naming consistency in the other direction.

Reply:
Why do you not like target? I'm fine either way.

Reply:
Wanna actually add the one above and do the same thing you did for y? The tag is not used in the estimator or tags yet, I think.

Reply:
> Why do you not like target? I'm fine either way.

Mostly to be consistent with `fit(X, y)`, but it's not too critical I agree.

> Wanna actually add the one above and do the same thing you did for y? The tag is not used in the estimator or tags yet, I think.

Sure will make another PR.